doc/script_commands.txt

```diff
@@ -7760,6 +7760,46 @@ scripts-atcommands this way.
 
 ---------------------------------------
 
+*checkpermission("<command>"{, <account id>})
```

Comment:
well they're both related to group permissions, and atcommand can only ever be string while perms can only ever be integer (from const)